src/OffGame/InitWidget.py

```diff
@@ -0,0 +1,33 @@
+import sys
+from PyQt5 import QtWidgets
+from UI_InitWidget import Ui_InitWidget
```

Comment:
Импортируй из модуля
```suggestion
from OffGame.UI_InitWidget import Ui_InitWidget
```